RecCalorimeter/src/components/CaloTowerTool.cpp

```diff
@@ -23,6 +23,9 @@ CaloTowerTool::CaloTowerTool(const std::string& type, const std::string& name, c
   declareProperty("hcalExtBarrelCells", m_hcalExtBarrelCells, "");
   declareProperty("hcalEndcapCells", m_hcalEndcapCells, "");
   declareProperty("hcalFwdCells", m_hcalFwdCells, "");
+  declareProperty("positionsECalBarrelTool", m_cellPositionsECalBarrelTool,
+                  "Handle for tool to retrieve cell positions in ECal Barrel");
```

Comment:
Is there a way to not introduce the dependency on the Cell Position tools?

Reply:
Do you need to adjust cell positions or you asume they are not positioned at all?

Reply:
I need to get the cell positions in theta and phi. For theta, we can also use "segmentation->theta(cellID)" from segmentation. However "segmentation->phi(cellID)" only returns a relative shift in phi, not the actual phi position of a cell. So @giovannimarchiori suggested me to use the Cell Position Tool.
What do you think? @giovannimarchiori

Reply:
The cell positions are needed to find out to which tower in eta-phi they belong, but the position does not need to be modified. So probably one could just run the clustering on the positioned cells and there would be no need for the positioning tool here.
@dasphy this means that you should use "ECalBarrelPositionedCells" rather than EcalBarrelCellsName (= "ECalBarrelCells") for towers.ecalBarrelCells.Path in https://github.com/BrieucF/LAr_scripts/pull/16/files#diff-58bdd8a8f3d731a4b580bf79c17c026b7ccd59c296d01a3c1aa01b02ffd6a4cf
Can you try to use the positioned cells (and retrieve theta/phi directly from their position) rather than from the positioning tool or the segmentation?

Reply:
That's a good idea, thanks @giovannimarchiori